packages/core/src/secret/bip32.ts

```
@@ -1,7 +1,10 @@
+/* eslint-disable spellcheck/spell-checker */
 // eslint-disable-next-line max-classes-per-file
 import BigNumber from 'bignumber.js';
 
-import { hmacSHA512 } from './hash';
+import { OneKeyPlainTextError } from '@onekeyhq/shared/src/errors';
+
+import { hmacSHA512, hmacSHA512Sync } from './hash';
```

ESLint

[error] hmacSHA512 not found in './hash'

(import/named)